Cargo.toml

```diff
@@ -203,13 +203,12 @@ zksync_protobuf = { version = "0.1.0", git = "https://github.com/matter-labs/era
 zksync_protobuf_build = { version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "3e6f101ee4124308c4c974caaa259d524549b0c6" }
 
 # "Local" dependencies
-multivm = { path = "core/lib/multivm" }
-prometheus_exporter = { path = "core/lib/prometheus_exporter" }
-prover_dal = { path = "prover/prover_dal" }
-vlog = { path = "core/lib/vlog" }
-vm_utils = { path = "core/lib/vm_utils" }
-vm-benchmark-harness = { path = "core/tests/vm-benchmark/harness" }
-zksync = { path = "sdk/zksync-rs" }
+zksync_multivm = { path = "core/lib/multivm" }
+zksync_prometheus_exporter = { path = "core/lib/prometheus_exporter" }
```

Comment:
Off-topic: Looks like we finally can remove  the `metrics` crate from the workspace dependencies. I'll tackle this separately.